quint/src/types/constraintGenerator.ts

```diff
@@ -310,6 +327,24 @@ export class ConstraintGeneratorVisitor implements IRVisitor {
     const subs = compose(this.table, typeSubs, rowSubs)
     return applySubstitution(this.table, subs, t.type)
   }
+
+  private currentFreeNames(): { typeVariables: Set<string>; rowVariables: Set<string> } {
+    return (
+      this.freeNames[this.freeNames.length - 1] ?? {
+        typeVariables: new Set(),
+        rowVariables: new Set(),
+      }
+    )
+  }
+
+  private quantify(type: QuintType): TypeScheme {
```

Comment:
Is quantification a common name for this operation?

Reply:
Yes, I believe so.

To be sure is not something my advisor came up with and I kept using, I checked and there are mentions of quantification for this sort of operation in the [Hindley-Milner type system wikipedia page](https://en.wikipedia.org/wiki/Hindley%E2%80%93Milner_type_system).